FRANKDOC.md

```diff
@@ -9,6 +9,7 @@ The Frank!Doc provides reference information for Frank developers as explained i
 | JavaDoc tag | `@ff.parameters` | Class | Describes how parameters (Frank config `<Param>`) are used. |
 | JavaDoc tag | `@ff.parameter` | Class | Describes the meaning of a specific parameter. First argument is name of parameter. Second argument is description of that parameter.|
 | JavaDoc tag | `@ff.forward` | Class | Describes a forward (e.g. `success`, `failure`). First argument is name of forward. Second argument is description.|
+| JavaDoc tag | `@ff.tag` | Class | Tag that classifies the Java class on behalf of the Frank!Doc webapp. First argument is tag name, second argument is tag value.|
```

Comment:
Het is niet 'on behalf of the Frank!Doc webapp'.

Reply:
"on behalf of" ==> "for". Is dat beter? We tonen dit niet in de XSD; dat lijkt me relevant om hier op te merken.

Reply:
Je zou kunnen zeggen dat tags 'bijvoorbeeld gebruikt kunnen gaan worden in de webapp om te filteren'.

Reply:
Gerrit, ik heb het doel van de klassificatie eruit gehaald, zoals besproken aan de telefoon.